test/unit/component/parser/json/BpmnJsonParser.task.receive.test.ts

```diff
@@ -17,61 +17,26 @@ import { ShapeBpmnElementKind } from '../../../../../src/model/bpmn/shape/ShapeB
 import { parseJsonAndExpectOnlyFlowNodes, verifyShape } from './JsonTestUtils';
 
 describe('parse bpmn as json for receive task', () => {
-  it('json containing one process with a single receive task', () => {
-    const json = `{
-                "definitions" : {
-                    "process": {
+  const processJsonAsObjectWithTaskJsonAsObject = `{
```

Comment:
:question: I don't understand why we have a dedicated file for `receive task` and why it is not put with other tasks test.

Reply:
Like I said in the description, there is a specific test for the receive task in order to know if it is _instantiate_ or not.
I'll refactor it in the next step with passing the attribute we want to test in the flowNode.
And I'll merge all tests for flownode in a single file (except events), because they are all almost the same.

Reply:
Yes, I saw that but 
- they are less regular tests for receive task than for other task tests
- why not putting regular tests with the other task tests and only keep specific tests in the dedicated receive task tests file

Reply:
Done 😉